tests/manage/test_dynamic_pvc_accessmodes_with_reclaim_policies.py

```diff
@@ -295,10 +282,158 @@ def finalizer():
+        pod_obj2.run_io(
+            storage_type=self.storage_type, size=self.io_size, runtime=30,
+            fio_filename=file_name2
+        )
+
+        # Check IO and calculate md5sum of files
+        pod.get_fio_rw_iops(pod_obj1)
+        md5sum_pod1_data = pod.cal_md5sum(
+            pod_obj=pod_obj1, file_name=file_name1
+        )
+
+        pod.get_fio_rw_iops(pod_obj2)
+        md5sum_pod2_data = pod.cal_md5sum(
+            pod_obj=pod_obj2, file_name=file_name2
+        )
+
+        logger.info(f"verify data from alternate pods")
+
+        assert pod.verify_data_integrity(
+            pod_obj=pod_obj2, file_name=file_name1,
+            original_md5sum=md5sum_pod1_data
+        )
+
+        assert pod.verify_data_integrity(
+            pod_obj=pod_obj1, file_name=file_name2,
+            original_md5sum=md5sum_pod2_data
+        )
+
+        # Verify that data is mutable from any pod
+
+        logger.info(f"Perform modification of files from alternate pod")
+        # Access and rename file written by pod-2 from pod-1
+        file_path2 = pod.get_file_path(pod_obj2, file_name2)
+        logger.info(file_path2)
+        pod_obj1.exec_cmd_on_pod(
+            command=f"bash -c \"mv {file_path2} {file_path2}-renamed\"",
+            out_yaml_format=False
+        )
+
+        # Access and rename file written by pod-1 from pod-2
+        file_path1 = pod.get_file_path(pod_obj1, file_name1)
+        logger.info(file_path1)
+        pod_obj2.exec_cmd_on_pod(
+            command=f"bash -c \"mv {file_path1} {file_path1}-renamed\"",
+            out_yaml_format=False
+        )
+
+        logger.info(f"Verify presence of renamed files from both pods")
+        file_names = [f"{file_path1}-renamed", f"{file_path2}-renamed"]
+        for file in file_names:
+            assert pod.check_file_existence(pod_obj1, file), (
+                f"File {file} doesn't exist"
+            )
+            logger.info(f"File {file} exists in {pod_obj1.name} ")
+            assert pod.check_file_existence(pod_obj2, file), (
+                f"File {file} doesn't exist"
+            )
+            logger.info(f"File {file} exists in {pod_obj2.name}")
+
+        pod_obj1.delete()
+        pod_obj1.ocp.wait_for_delete(resource_name=pod_obj1.name)
+
+        pod_obj2.delete()
+        pod_obj2.ocp.wait_for_delete(resource_name=pod_obj2.name)
+
+
+@tier3
+@pytest.mark.usefixtures(
+    create_ceph_block_pool.__name__,
+    create_rbd_secret.__name__,
+)
+@pytest.mark.parametrize(
+    argnames=["interface_type", "reclaim_policy"],
+    argvalues=[
+        pytest.param(
+            *[constants.CEPHBLOCKPOOL, constants.RECLAIM_POLICY_RETAIN],
+            marks=pytest.mark.polarion_id("OCS-547")
+        ),
+        pytest.param(
+            *[constants.CEPHBLOCKPOOL, constants.RECLAIM_POLICY_DELETE],
+            marks=pytest.mark.polarion_id("OCS-538")
+        )
+
+    ]
+)
+class TestRWXDynamicPvcRBD(BaseDynamicPvc):
```

Comment:
Please try to combine this class along with TestRWXDynamicPvc, into one class with all the possible combinations for FS and RBD, using pytest parametrize

Reply:
@elad we tried to combine the two together, but since PVC creation itself is an expected failure for  rwx+ rbd , we would need to add the check in setup_base itself. Will try to find a better solution

Reply:
@ebenahar combined the two. When Parikshit will send the code for "RWX +RBD" ,  he will be handling the pv creation failure.

Reply:
Great!